cypress/support/fragments/settings/finance/settingsFinance.js

```diff
@@ -42,7 +42,7 @@ export default {
     cy.do(MultiColumnListCell({ content: expenseClass.name }).perform(
       element => {
         const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
-        const createdByAdmin = `${DateTools.getFormattedDateWithSlashes({ date: new Date() })} by ADMINISTRATOR, DIKU`;
```

Comment:
Please use more universal approach. Take the Last name[, First name] from the profileDropdown

Reply:
![image](https://user-images.githubusercontent.com/17912474/201284807-1c33557d-d892-43a0-8fef-e6fe8793b4fd.png)